InfiniteRecharge/src/main/java/frc/robot/subsystem/scoring/shooter/ShooterSubsystem.java

```diff
@@ -12,6 +12,9 @@
 import frc.robot.utils.math.MathUtils;
 import frc.robot.utils.talonutils.MotorUtils;
 
+import frc.robot.utils.data.filters.RunningAverageFilter;
+import frc.robot.subsystem.drive.DriveConstants;

```

Comment:
once you put it in ShooterConstants make sure to get rid of this (what will be) unused import